src/index.json

```diff
@@ -1290,8 +1290,8 @@
                     "license": "MIT",
                     "metadata_version": "2.0",
                     "name": "front-door",
-                    "summary": "Manage networking FrontDoors.",
-                    "version": "0.1.0"
+                    "summary": "Manage networking Front Doors.",
+                    "version": "0.1.1"
                 }
             }
```

Comment:
Are you publishing express routes too?

Reply:
It is already published. The Front Door extension needed a version bump because the help text changed.